.github/workflows/performance-caller.yml

```
@@ -0,0 +1,74 @@
+
+      # Baseline
+      baseline-file: 'docs/benchmarks/baselines/phase3_student_cpu.json'
+
+      # Execution
+      python-version: '3.12'
+      warmup-iterations: 10
+      benchmark-iterations: 100
+
+      # Synthetic data generation
+      generate-synthetic-data: true
+      synthetic-data-script: |
```

actionlint (1.7.10)

property "test-data-directory" is not defined in object type {}

(expression)